cloud/azure/sp-azure.tf

```diff
@@ -0,0 +1,31 @@
+resource "azuread_application" "signalfx_integration" {
+  name     = "SignalFxAzureIntegration"
```

Comment:
Last nitpick: what about using the same name as for the integration?
https://github.com/claranet/terraform-signalfx-integrations/pull/1/files#diff-87343270e6da3b401095e3e210d0bfe3R2

PS: I thought I included this comment in my previous but apparently not.